api/routes.js

```diff
@@ -147,12 +147,12 @@ exports.set = function(app, mongo) {
 
     app.get(URL_SALT, (request, response) => {
         adminController.getByCriteria({ nickname : request.params.nickname }).then((users) => {
-            let user = users[0];
+            let user = users;
```

Comment:
is there a reason for not using `users` directly? :thinking:

Reply:
It's personal, but I don't like to use array everytime

Reply:
it's personal, I don't like to use array for everything